src/clients/firstboot_setup_dhcp.rb

```diff
@@ -1,6 +1,3 @@
-require "yast"
-require "network/network_autoconfiguration"
+require "network/clients/inst_setup_dhcp"
 
-Yast::NetworkAutoconfiguration.instance.configure_dhcp
-
-:next
+Yast::SetupDhcp.instance.main
```

Comment:
Now the dhcp configuration is skipped in case that the network is not handled by wicked, https://github.com/yast/yast-network/blob/master/src/lib/network/clients/inst_setup_dhcp.rb#L36 as we do during an installation.